test/AspNetCoreModule.Test/Framework/TestWebSite.cs

```diff
@@ -292,6 +292,24 @@ public TestWebSite(IISConfigUtility.AppPoolBitness appPoolBitness, string logger
                         TestUtility.LogInformation("Failed to overwrite project file, update the project file manually");
                     }
 
+
+                    if (TestFlags.Enabled(TestFlags.UseDotNetCore22))
+                    {
+                        string aspNetCoreModuleHostingModel, buildConstants;
+
+                        if (TestFlags.Enabled(TestFlags.InprocessMode))
+                        {
+                            aspNetCoreModuleHostingModel = "inprocess";
```

Comment:
Why do you need two of these? Can't you reused the same one?

Reply:
It's the same file. `aspNetCoreModuleHostingModel` is a build parameter

Reply:
Can you consolidate build parameters into one parameter?

Reply:
`aspNetCoreModuleHostingModel` is used here 
https://github.com/aspnet/AspNetCoreModule/blob/10637ae513fadd27c4dc33cc5d5bcdf2ab9e5d4c/test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj.UseDotNetCore22#L8
and consumed by IISIntegration
`BuildConstants` is used here
https://github.com/aspnet/AspNetCoreModule/blob/10637ae513fadd27c4dc33cc5d5bcdf2ab9e5d4c/test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj.UseDotNetCore22#L9
and used as a compiler directive to determine to call "UseIIS" or "UseIISIntegration"

Reply:
Can't really separate them as they are used by different consumers